api/src/main/java/io/opentelemetry/metrics/DefaultMeter.java

```diff
@@ -102,9 +102,9 @@ public static Meter getInstance() {
   }
 
   @Override
-  public BatchRecorder newBatchRecorder(LabelSet labelSet) {
+  public RecordBatch newRecordBatch(LabelSet labelSet) {
```

Comment:
RecordBatch is more like an action and not name of a class that is used to record a batch. Not sure if we want to call this RecordBatch. @jkwatson what do you think?

Reply:
I agree. A BatchRecorder is a thing that records batches. So the verb should be `recordBatch`, but the noun (object) is a `BatchRecorder` as we have now.

Reply:
I also prefer `BatchRecorder` as a name to use. It is not the specification name but it makes the code more *fluent* to read. 
@carlosalberto WDYT?

Reply:
A RecordBatch could also be a batch of records (returned by a BatchRecorder). I'd also prefer sticking to `BatchRecord.recordBatch()`.

Reply:
Not sure if I would duplicate the batch word in the class name and method name, I think just `record` is fine.

Reply:
how about `newRecorder` ?  The semantics are (I think) that a new recorder is being created, so having new in the name seems helpful.

Reply:
@jkwatson we currently have `newBatchRecorder`, I think the name is good, do you suggest to change that to just `newRecorder`? I am worried that this will become the official way to record things because recorder is very generic name.

Reply:
I'm ok with keeping it as-is with `newBatchRecorder` .

Reply:
I will close the issue then ;)